libpod/runtime_test.go

```diff
@@ -19,10 +18,8 @@ func Test_generateName(t *testing.T) {
 	}
 
 	// Test that (*Runtime).generateName returns different names
-	// if called twice, even if the global RNG has the default
-	// seed.
+	// if called twice.
 	n1, _ := r.generateName()
-	rand.Seed(1)
```

Comment:
I am not sure that the old test worked as intended, but the new one would really not be testing the “default seed” behavior.

(It seems to me that the concern is that two Podman processes don’t just start generating the same names, significantly raising the risk of collisions. Hence the test that names generated “in default conditions” don’t match the ones with `Seed(1)`.)

Reply:
I think that depends on whether we end up parametrizing the name generator with a RNG; if we do, we can have a good test.

If not, the version in this PR is probably as good as it gets, I agree.